src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -283,8 +284,11 @@ def complete(self, jobs):
                         condorFilePath = os.path.join(job['cache_dir'], condorFile)
                         if os.path.isfile(condorFilePath):
                             logTail = BasicAlgos.tail(condorFilePath, 50)
-                            logOutput += 'Adding end of %s to error message:\n' % condorFile
-                            logOutput += '\n'.join(logTail)
+                            logOutput += 'Adding end of %s to error message:\n\n' % condorFile
```

Comment:
I refer to L287. After that line, we get the log tail and it would be better not to put a new line between the filename and its content to sort of create blocks of logs in wmstats